blog-post.html

```diff
@@ -1,12 +1,51 @@
+
+
+
 <!DOCTYPE html>
 <html>
   <head>
     <title>Blog Post</title>
+    <link href="b-p-style.css" rel="stylesheet"/>
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Lexend+Giga&display=swap" rel="stylesheet"/>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
   </head>
+<html>
+
   <body>
-    <section id="header"></section>
-    <section id="main"></section>
-  </body>
+        <div id="margin1">to delete</div>  <!--margins of the body-->
+        <div id="margin2">to delete</div>
+        <section id="header"><div id="title">A Blog Post</div><div id="take">Complete with the Same Picture From the Other Pages<hr><hr></div><div id="subtitle">The Mountains Are Pretty Beautiful Though<br><br> <!--main content of the page-->

```

Comment:
One of the stories that we were looking for was having the headers be links back to the home page. We can do that with a simple `<a href="./index.html">`